libs/design/TableSkeleton/TableSkeleton.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-shadow */
+
+import { TableCell } from '../Table/TableCell';
+import { TableRow } from '../Table/TableRow';
+
+export const TableSkeleton = () => {
+  const numberOfSkeletonRows = 10;
+  const numberOfSkeletonCells = 4;
```

Comment:
It's better to take cols and rows from props.

Reply:
it shuold be static anyway cuase it's not working with table object.